rts/lra/lra-proxy/api/src/main/java/io/narayana/lra/client/internal/proxy/ParticipantProxy.java

```diff
@@ -88,19 +88,19 @@ void setFuture(Future<Void> future, boolean compensate) {
         this.compensate = compensate;
     }
 
-    private CompensatorStatus getExpectedStatus() {
-        return compensate ? CompensatorStatus.Compensated : CompensatorStatus.Completed;
+    private LRAStatus getExpectedStatus() {
```

Comment:
We now have separate status types for participants and LRAs (ParticipantStatus and LRAStatus). So the return here should be ParticipantStatus. There are lots of other places in the PR where we need to be returning ParticipantStatus and not LRAStatus.

Reply:
thanks! this was pretty wrong